_i18n/en.yml

```diff
@@ -102,98 +102,143 @@ readtime:
-  categories:
-    getting-started:
-      title: "Getting started"
-      intro: "Welcome to AntennaPod! Below, you can find some details on how to get started with AntennaPod."
-      subscribe: "Subscribing to a podcast"
-      home: "Using the Home screen"
-    general:
-      title: "General"
-      intro: "Below, you can find more details about the AntennaPod project."
-      ads: "About advertisements in AntennaPod"
-      backup: "Backing up your episodes"
-      beta: "Beta testing"
-      central-distributed: "Central and distributed podcast apps"
-      external-storage: "Using external storage"
-      synchronization: "How to synchronize between devices"
-      proxy: "Leading traffic through a proxy or Tor network"
-      episode-deletion: "Deletion of episodes"
-      f-droid: "Updates on F-Droid"
-      other-platforms: "AntennaPod for Windows, Apple and Linux"
-    playback:
-      title: "Playback"
-      intro: "Below, you can find topics related to playback."
-      auto-rewind: "Automatic rewind"
-      shortcuts: "Hardware keyboard shortcuts"
-      android-auto: "Playback in cars (Android Auto)"
-      chromecast: "Playback on Google Chromecast"
-    queue:
-      title: "Queue"
-      intro: "Below, you can find frequently asked questions about the queue."
-      how-to-use: "How to use the queue"
-      lock: "(Un)locking the queue"
-      smart-shuffle: "How to use smart shuffle"
-      multiple-queues: "How to use multiple queues"
-    subscriptions:
-      title: "Subscriptions"
-      intro: "Below, you can find frequently asked questions about subscriptions."
-      subscription-groups: "Subscription tags (aka groups or categories)"
-    automation:
-      title: "Automation"
-      intro: "Are you an experienced user and want to automate tasks? Have a look at the pages below."
-      deletion: "Automatic deletion"
-      refreshing-podcasts: "Refreshing podcasts"
-      tasker: "Integration with other apps"
-    bugs-first-aid:
-      title: "Bugs, errors & first aid"
-      intro: "Making software is a human craft, so mistakes are inevitable. But there's often something we can do about it!"
-      bug-report: "Something isn't working"
-      typo-translations: "I found a typo or spelling mistake"
-      database-error: "All my podcasts & episodes are gone"
-      cannot-stream-download: "Cannot download an episode"
-      episodes-missing: "Old episodes are missing"
-      playback-stops: "Playback suddenly stops"
-      refresh-download-errors: "Download or feed refresh error"
-      cannot-resume-playback: "Can't resume playback from lock screen / notification"
-    podcasters-hosters:
-      title: "For Podcasters & Hosters"
-      intro: "Below, you can find our support information for podcasters & hosting providers."
-      add-on-antennapod: "Create an 'Open in AntennaPod' link"
-      list-podcast: "Getting your podcast listed"
-      update-feed-urls: "Updating feed URLs"
+  getting-started:
```

Comment:
What happens with the translations after this restructuring? Do they all need to be re-done?

Reply:
To be confirmed